component/src/main/java/com/synopsys/integration/alert/component/settings/proxy/database/accessor/SettingsProxyConfigAccessor.java

```diff
@@ -93,9 +87,12 @@ public SettingsProxyModel createConfiguration(SettingsProxyModel configuration)
 
     @Override
     @Transactional(propagation = Propagation.REQUIRED)
-    public SettingsProxyModel updateConfiguration(UUID configurationId, SettingsProxyModel configuration) throws AlertConfigurationException {
-        SettingsProxyConfigurationEntity configurationEntity = settingsProxyConfigurationRepository.findById(configurationId)
-                                                                   .orElseThrow(() -> new AlertConfigurationException(String.format("Config with id '%s' did not exist", configurationId.toString())));
+    public SettingsProxyModel updateConfiguration(SettingsProxyModel configuration) throws AlertConfigurationException {
+        SettingsProxyConfigurationEntity configurationEntity = settingsProxyConfigurationRepository.findByName(configuration.getName())
```

Comment:
This may be confusing to a new user. If they try updating the hidden name using our API it would throw this exception. We may want to stick to the default of using the default name